src/rest_framework_jwt/blacklist/models.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+
+from django.conf import settings
+from django.db import models
+
+
+class BlacklistedTokenManager(models.Manager):
+    def delete_stale_tokens(self):
+        return self.filter(expires_at__lt=datetime.utcnow()).delete()
```

Comment:
Shouldn't this be handled via a setting? For instance, delete all tokens that are older than the specified delta?